docs/domains/Domain.json

```diff
@@ -51,6 +51,14 @@
           "items": {
             "$ref": "#/definitions/Channel"
           }
+        },
```

Comment:
Just a heads-up that we can't merge this until we merge the code to upgrade the Domain CRD version.  Even though you would just be adding optional fields (which is good), the issue is that unless we version the CRD then a 2.0.1 operator would revert the change to the CRD by writing its version.